src/braket/experimental/autoqasm/types/conversions.py

```diff
@@ -52,6 +52,8 @@ def map_type(python_type: type) -> type:
         # ctx = program.get_program_conversion_context()
         # dims = ctx.get_oqpy_program().declared_vars[name_of_var].dimensions
         return oqpy.ArrayVar[oqpy.IntVar, 10]
+    if issubclass(origin_type, tuple):
+        raise TypeError('Tuple parameter types are not supported; please use "list" instead.')
```

Comment:
Just a suggestion here to make this messaging more clear - feel free to change this further.
```suggestion
        raise TypeError('Tuples are not supported as parameters to AutoQASM functions; please separate the tuple into multiple parameters or use a list instead.')
```